.chronus/changes/logEnhance-2025-2-26-10-22-33.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: internal
```

Comment:
this is not internal, this actually fix an issue that is relevant to the end user

Reply:
also should update the message to be more descriptive for the user of the extension of what this fixed